docker/dev/lemmy-dev.hjson

```diff
@@ -0,0 +1,61 @@
+{
```

Comment:
Looks like this file still needs to be deleted. Otherwise looks good.

Reply:
Yes.